api/go.mod

```diff
@@ -11,9 +11,9 @@ require (
 	github.com/openstack-k8s-operators/lib-common/modules/openstack v0.4.1-0.20240910122823-bdcfe881513b
 	github.com/openstack-k8s-operators/lib-common/modules/test v0.4.1-0.20240910122823-bdcfe881513b
 	golang.org/x/exp v0.0.0-20240719175910-8a7402abbf56
-	k8s.io/api v0.28.13
-	k8s.io/apimachinery v0.28.13
-	sigs.k8s.io/controller-runtime v0.16.6
+	k8s.io/api v0.29.2
+	k8s.io/apimachinery v0.29.2
+	sigs.k8s.io/controller-runtime v0.17.5
```

Comment:
Do we want to do latest 0.17.x everywhere:
https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.17.6

Reply:
I wanted to have something to verify that renovate to bump things properly